lib/parlour/rbi_generator/namespace.rb

```diff
@@ -285,32 +288,38 @@ def create_attribute(name, kind:, type:, &block)
       # @param name [String] The name of this attribute.
       # @param type [String] A Sorbet string of this attribute's type, such as
       #   +"String"+ or +"T.untyped"+.
+      # @param class_attribute [Boolean] Whether this attribute belongs to the
+      #   singleton class.
       # @param block A block which the new instance yields itself to.
       # @return [RbiGenerator::Attribute]
-      def create_attr_reader(name, type:, &block)
-        create_attribute(name, kind: :reader, type: type, &block)
+      def create_attr_reader(name, type:, class_attribute: false, &block)
+        create_attribute(name, kind: :reader, type: type, class_attribute: class_attribute, &block)
       end
 
       # Creates a new write-only attribute (+attr_writer+).
       #
       # @param name [String] The name of this attribute.
       # @param type [String] A Sorbet string of this attribute's type, such as
       #   +"String"+ or +"T.untyped"+.
+      # @param class_attribute [Boolean] Whether this attribute belongs to the
+      #   singleton class.
       # @param block A block which the new instance yields itself to.
       # @return [RbiGenerator::Attribute]
-      def create_attr_writer(name, type:, &block)
-        create_attribute(name, kind: :writer, type: type, &block)
+      def create_attr_writer(name, type:, class_attribute: false, &block)
+        create_attribute(name, kind: :writer, type: type, class_attribute: class_attribute, &block)
       end
 
       # Creates a new read and write attribute (+attr_accessor+).
       #
       # @param name [String] The name of this attribute.
       # @param type [String] A Sorbet string of this attribute's type, such as
       #   +"String"+ or +"T.untyped"+.
+      # @param class_attribute [Boolean] Whether this attribute belongs to the
+      #   singleton class.
       # @param block A block which the new instance yields itself to.
       # @return [RbiGenerator::Attribute]
-      def create_attr_accessor(name, type:, &block)
-        create_attribute(name, kind: :accessor, type: type, &block)
+      def create_attr_accessor(name, type:, class_attribute: false, &block)
```

Comment:
Maybe a good time to add sigs for `create_attr_accessor`, `create_attr_reader`, `create_attr_writer`, and `create_attribute`? :)